### PR TITLE
Simplify webdav templates

### PR DESCRIPTION
**Pull request by basilelegal**
Rework of templates for webdav files thread and page

### CODE REVIEW

File: backend/tracim_backend/lib/webdav/design.py [hunk 1]
```diff
@@ -124,6 +126,16 @@
...
 
+THREAD_MESSAGE = """
+  <p>
+    {posting_time},{comment_owner} wrote :
+  </p>
+  <p>
+    {comment_content}
+  </p>
+
+"""
```
Comment (tracim) [inline backend/tracim_backend/lib/webdav/design.py]: @basilelegal, the way to show indented messages is:

- to write "the xxx, basile wrote:"
- to indent the content

In your MR, you put introduction message (the xxx, basile wrote:) and the user message at same indentation level, which is wrong.

Example taken from an existing email:

```
Le 14/12/2018 à 14:27, michel a écrit :
    Bonjour Damien,
    [...]
```

Comment (basilelegal) [reply backend/tracim_backend/lib/webdav/design.py]: Done

File: backend/tracim_backend/lib/webdav/design.py [hunk 2]
```diff
@@ -124,6 +126,16 @@
     'copy' : 'Item copied',
 }
 
+THREAD_MESSAGE = """
+  <p>
+    {posting_time},{comment_owner} wrote :
+  </p>
+  <p style="padding-left: 1em;">
```
Comment (tracim) [inline backend/tracim_backend/lib/webdav/design.py]: @basilelegal why dont you use the same design as ou do for other messages: `<blockquote style="border-left: solid 2px #999; margin-left: 1em; padding-left: 1em;">` ?

File: backend/tracim_backend/lib/webdav/design.py [hunk 3]
```diff
@@ -124,6 +126,16 @@
     'copy' : 'Item copied',
 }
 
+THREAD_MESSAGE = """
+  <p>
+    {posting_time},{comment_owner} wrote :
```
Comment (tracim) [inline backend/tracim_backend/lib/webdav/design.py]: by the way, there are no space before a colon: `Michel wrote :` => `Michel wrote:` (space removed before `:`)

Comment (basilelegal) [reply backend/tracim_backend/lib/webdav/design.py]: Done

**Pull request merged**